File: example_assignments/cs121-moravian-java/hw02-simple_stats/base_code/src/SimpleStats.java
public class SimpleStats {
    private int count = 0;
    private double sum = 0.0;

    public void addValue(double newValue) {
        sum += newValue;
        count += 1;
    }

    public int getCount() {
        return count;
    }

    public double getAverage() {
        if (count == 0) {
            return 0.0;
        }
        else {
            return sum / count;
        }
    }
}

File: example_assignments/cs120-moravian-python/hw19-simple_stats/solution/simple_stats.py
class SimpleStats:
    def __init__(self):
        self.number_list = []

    def add_value(self, value):
        self.number_list.append(value)

    def get_min(self):
        return min(self.number_list)

    def get_max(self):
        return max(self.number_list)

    def get_sum(self):
        return sum(self.number_list)

    def get_average(self):
        if len(self.number_list) == 0:
            return 0

        return sum(self.number_list) / len(self.number_list)

    
stats = SimpleStats()

new_number = input('Enter a number for the list: ')

while new_number != '':
    stats.add_value(eval(new_number))
    new_number = input('Enter a number for the list: ')

print('The smallest number is', stats.get_min())
print('The largest number is', stats.get_max())
print('The sum of the list is', stats.get_sum())
print('The average of the list is', stats.get_average())
